internal/resource/obtenantrestore/obtenantrestore_flow.go

```diff
@@ -17,11 +17,11 @@ import (
 	tasktypes "github.com/oceanbase/ob-operator/pkg/task/types"
 )
 
-func StartRestoreJob() *tasktypes.TaskFlow {
+func FlowStartRestoreJob() *tasktypes.TaskFlow {
```

Comment:
1. is it necessary to add Flow as prefix
2. is it possible to pass the resource manager or the certain resource to generate different task flow based on resource's metadata e.g. cluster mode annotation